openapi/main.yaml

```diff
@@ -0,0 +1,704 @@
+              schema:
+                type: object
+                properties:
+                  transactionxdrs: 
+                    type: array
+                    items:
+                      type: string
+                      description: base64 encoded transaction xdr string
+              example:
+                transactionxdrs:
+                  - "Y6MF7SMT2a2d6pt3i37Xx9"
+                  - "M8GA7SMT2a2d6pt3i37Xx8"
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                  error:
+                    type: string
+                    description: Details about the error
+                example:
+                  status: 400
+                  error: Bad operation xdr.
+        '500':
+          description: Internal Server Error
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                  error:
+                    type: string
+                example:
+                  status: 500
+                  error: An error occurred while processing this request.
+              example:
+                status: 500
+                error: An error occurred while processing this request.
+  /tss/transactions:
+    post:
+      tags:
+        - TSS
+      summary: submit transactions in bulk to TSS
+      description:
+        Submit a list of transactions to TSS so that it can submit it to the stellar network
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                webhookurl:
+                  type: string
+                  description: The url to which the final transaction results are to be sent
+                transactions:
+                  type: array
+                  items:
+                    type: string
+                    description: base64 encoded transaction xdr to submit to the stellar network
+                feebump:
+                  type: boolean
+                  description: boolean indicating if this transaction should be wrapped in a fee bump transaction
+              required:
+                - webhookurl
+                - transactions
+            example:
+              webhookurl: "www.sdp.stellar.com/webhook"
+              transactions:
+                - "Y6MF7SMT2a2d6pt3i37Xx9"
+                - "M8GA7SMT2a2d6pt3i37Xx8"
+              feebump: true
+      responses:
+        '200':
+          description: "Successful response with the transaction hashes of the transactions to be submitted to the network. Clients can use these hashes to query TSS for the status of their transaction"
```

Comment:
Does this endpoint submit all transactions provided in the request to RPC before returning? Or are they just queued for submission?

Reply:
Nope. Transactions are queued for submission. These transaction hashes are just receipts that the client can use to call the GET /tss/transactions/{txhash} endpoint